crates/cli-tools/src/changelog.rs

```diff
@@ -212,6 +261,20 @@ impl Release {
     fn new(version: &str) -> Result<Self> {
         Ok(Release { version: Version::parse(version)?, contents: BTreeMap::new() })
     }
+
+    fn push_content(&mut self, severity: Severity, content: &str) -> Result<()> {
+        let entry = self.contents.entry(severity).or_default();
+
+        entry.push(content.to_string());
+
+        Ok(())
```

Comment:
nit: no need for so much vertical space

```suggestion
        self.contents.entry(severity).or_default().push(content.to_string());
        Ok(())
```